python_docs_theme/layout.html

```diff
@@ -38,11 +38,71 @@
 {% block extrahead %}
     <link rel="shortcut icon" type="image/png" href="{{ pathto('_static/' + theme_root_icon, 1) }}" />
     {% if builder != "htmlhelp" %}
-    {% if not embedded %}<script type="text/javascript" src="{{ pathto('_static/copybutton.js', 1) }}"></script>{% endif %}
+        {% if not embedded %}
+            <script type="text/javascript" src="{{ pathto('_static/copybutton.js', 1) }}"></script>
+            <script type="text/javascript" src="{{ pathto('_static/menu.js', 1) }}"></script>
+        {% endif %}
     {% endif %}
     {{ super() }}
 {% endblock %}
 
+{%- block css %}
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
This got added in sphinx 3.1 and should work mostly without problem for people using old sphinx versions. May however not hurt to include it for now and delete it latex

Reply:
I added this to sphinx after creating this PR.